test/cases/base_test.rb

```diff
@@ -0,0 +1,31 @@
+require "cases/helper_cockroachdb"
+
+module CockroachDB
+  class BasicsTest < ActiveRecord::TestCase
+    # This replaces the same test that's been excluded from BasicsTest. It's
+    # exactly the same, except badchar has an entry for CockroachDBAdapter.
+    def test_column_names_are_escaped
+      conn      = ActiveRecord::Base.connection
+      classname = conn.class.name[/[^:]*$/]
+      badchar   = {
+        "SQLite3Adapter"    => '"',
```

Comment:
If this lookup table exist in the base, should the entry for CRDB be there?

Reply:
Good question! The lookup table is [in the ActiveRecord test suite](https://github.com/rails/rails/blob/a0fb2b2227c7e9eacceac65b40c8a59dbfb497ec/activerecord/test/cases/base_test.rb#L100-L108), but I'm not sure they'll accept a PR to add CockroachDB since it's not an officially supported adapter.